engine/gamesys/proto/gamesys/gui_ddf.proto

```diff
@@ -155,7 +155,7 @@ message NodeDesc
     optional bool enabled                   = 45 [default = true];
     optional bool visible                   = 46 [default = true];
 
-    optional string material                = 47 [(resource)=true];
```

Comment:
If the field is a resource, then the bundler uses it as a path and tries to find such path in build folder.
But Material field on a node, just a string we use to know what material we should use from GUI materials folder.